downstream/modules/hub/proc-create-content-developers.adoc

```diff
@@ -2,17 +2,17 @@
 
 = Creating a new group for content curators
 
-You can create a new group in {PrivateHubName} designed to support content curation in your organization. This group can contribute internally developed collections for publication in {PrivateHubName}.
+You can create a new group in {PlatformNameShort} designed to support content curation in your organization. This group can contribute internally-developed collections for publication in {PrivateHubName}.
 
 To help content developers create a namespace and upload their internally developed collections to {PrivateHubName}, you must first create and edit a group and assign the required permissions.
 
 .Prerequisites
 
-* You have administrative permissions in {PrivateHubName} and can create groups.
+* You have administrative permissions in {PlatformNameShort} and can create groups.
 
 .Procedure
 //[ddacosta] For 2.5 this will be Log in to Ansible Automation Platform and select Automation Content. Automation hub opens in a new tab. From the navigation ...
-. Log in to your {PrivateHubName}.
+. Log in to your {PlatformNameShort}.
 . From the navigation panel, select {MenuHubGroups} and click btn:[Create].
```

Comment:
This procedure is completely different now:

. From the navigation panel, select {MenuAMTeams} and click btn:[Create teams].
. Enter *Content Engineering* as a *Name* for the team and optionally provide a *Description*. 
. Select an organization from the *Organization* list and click btn:[Create team].
. You have created the new team and the *Teams* page is displayed.
. Select the *Roles* tab and then select the *Automation Content* tab. 
. Click btn[Add roles].
. Select *Namespace* from the *Resources type* list.
. Select the namespaces that will receive new roles and click btn:[Next].
. Select the roles to be applied to the selected namespaces and click btn:[Next].
. Review the settings and click btn:[Finish].

The new team includes the permissions that you assigned. Now, you can add users to the team.

. Click the *Users* tab on the *Teams* page.
. Click btn:[Add users].
. Select users and click btn:[Add users].

Reply:
I think I see what happened here. I did a ctrl+f search for automation hub and replaced those attributes with the one for AAP, so that's why this module is showing up in the PR. But, I had commented it out in assembly-working-with-namespaces.adoc with the intent of linking to your Access Management and Authentication doc, so I didn't update the procedure. Argh, my mistake, apologies Donna.

Reply:
I didn't update this procedure since I commented it out of the draft and instead linked to the Access Management and Authentication draft. If you think it's a better idea, I can add it back in--it looks like the steps you suggested above are a little more specific to content engineering groups. LMK what you think.

Reply:
@hherbly  I'm totally fine with your pointing to the Access management guide for creating teams and users. If this module isn't used anywhere, you can ignore my suggestions.